src/org/daisy/dotify/formatter/impl/sheet/SheetDataSource.java

```diff
@@ -202,8 +201,8 @@ public void setCurrentVolumeNumber(int volume) {
      * @return returns true if the index element was available, false otherwise
      */
     private boolean ensureBuffer(int index) {
-        Sheet.Builder s = null;
-        SheetIdentity si = null;
+        Sheet.Builder s = null; // sheet currently being built
+        SheetIdentity si = null; // sheet currently being built
```

Comment:
This is not strictly correct, one is the ID of the sheet and the other is the builder, not important but I wonder if there is a programmatic reason to have both? Having multiple classes to hold the same state could create interesting concurrency issues. Could the Sheet class hold the identity?

If not this is fine, I will accept the PR so this is more of a question.

Reply:
Yes I guess the Sheet class could hold the identity and that could simplify the SheetDataSource code a bit. On the other hand, SheetIdentity is an internal something, for interfacing with CrossReferenceHandler, that does not need to be returned from SheetDataSource. So I'm unsure if including it in Sheet is really the best thing to do.